extensions/answer_summarizers/models.py

```diff
@@ -475,7 +475,14 @@ def calculate_from_state_answers_dict(
             str, List[state_domain.AcceptableCorrectAnswerTypes]
         ] = collections.defaultdict(list)
         for category, answer_dicts in grouped_submitted_answer_dicts:
-            if category in CLASSIFICATION_CATEGORIES:
+            # The 'classification_categorization' of SubmittedAnswer cannot
```

Comment:
If this is the case then either use an assertion or raise exception?

You really shouldn't be using "no branch" anywhere.

Reply:
Done